e2e-tests/upgrade-consistency/compare/statefulset_some-name-rs0-150.yml

```diff
@@ -2,6 +2,13 @@ apiVersion: apps/v1
 kind: StatefulSet
 metadata:
   generation: 5
+  labels:
```

Comment:
I don't think that labels should be added in versions less than 1.6.0

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/445/commits/c9ebe9e256d08f20ee32f29b015b3bdc50b225db